public/robots.txt

```diff
@@ -1,7 +1,14 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
-#
-# To ban all spiders from the entire site uncomment the next two lines:
-# User-agent: *
-# Disallow: /
+
+User-agent: tGooglebot
```

Comment:
@bklaing2 could this be a typo, the `t` at the beginning of the UserAgent

Reply:
Oops yes, thank you!